src/daemon/meson.build

```diff
@@ -51,6 +52,7 @@ daemon_deps = [
     dep_xfce4windowing,
     dep_gst,
     dep_cairo,
+    dependency('upower-glib', version: '>= 0.99.8'),
```

Comment:
This dep should be declared in the root meson.build and *referenced* in this file.

Reply:
true, true.  Same issue as the PowerInhibitor meson.build file.

We should fix the powerinhibitor deprecation API usage issues and meson.build in a separate PR to not complicate and confuse matters here.